frontend/views/components/tabs/TabWrapper.vue

```diff
@@ -34,6 +40,8 @@ import sbp from '@sbp/sbp'
 import { mapGetters } from 'vuex'
 import TabItem from '@components/tabs/TabItem.vue'
 import { logExceptNavigationDuplicated } from '@view-utils/misc.js'
+import { L } from '@common/common.js'
+import packageJSON from '~/package.json'
```

Comment:
I'm not sure, but eslint might be able to optimize this by not including the entire package.json file, and just the version number, if you import like this: `import { contractsVersion } from '~/package.json'`